src/pages/DetailsPage/DetailsPage.js

```diff
@@ -29,28 +27,32 @@ export default function DetailsPage() {
   return (
     <PageContainer>
       <BackButton />
-      <ContentWrapper>
-        <HeaderContainer>
-          <ImageContainer>
-            <Image src={image} alt={`Poster`} width="40px" height="40px" />
-          </ImageContainer>
-          <InnerContainer>
-            <Title>{title}</Title>
-            <Author>{author}</Author>
-          </InnerContainer>
-        </HeaderContainer>
-        <BodyContainer>
-          <FavorizeButton id={Number(id)} />
-          <Text>{description}</Text>
-          {categories && (
-            <TagList>
-              {Object.entries(categories).map(([key, value]) => (
-                <Tag key={key}>{value}</Tag>
-              ))}
-            </TagList>
-          )}
-        </BodyContainer>
-      </ContentWrapper>
+      {isLoading ? (
+        <LoadingScreen />
+      ) : (
+        <ContentWrapper>
+          <HeaderContainer>
+            <ImageContainer>
+              <Image src={image} alt={`Poster`} width="40px" height="40px" />
+            </ImageContainer>
+            <InnerContainer>
+              <Title>{title}</Title>
+              <Author>{author}</Author>
+            </InnerContainer>
+          </HeaderContainer>
+          <BodyContainer>
+            <FavorizeButton id={Number(id)} />
+            <Text>{description}</Text>
+            {categories && (
+              <TagList>
+                {Object.entries(categories).map(([key, value]) => (
+                  <Tag key={key}>{value}</Tag>
+                ))}
+              </TagList>
+            )}
+          </BodyContainer>
+        </ContentWrapper>
```

Comment:
This looks like it could've been taken in another component (Maybe even two components if it is possible to separate Header & Container). 
```suggestion
        <Details>
```

Details.js
```js
<ContentWrapper>
    <Header />
    <Body />
</ContentWrapper>
```

<customName>Header.js
```js
<HeaderContainer>
  <ImageContainer>
    <Image src={image} alt={`Poster`} width="40px" height="40px" />
  </ImageContainer>
  <InnerContainer>
    <Title>{title}</Title>
    <Author>{author}</Author>
  </InnerContainer>
</HeaderContainer>
```

<customName>Body.js
```js
<BodyContainer>
  <FavorizeButton id={Number(id)} />
  <Text>{description}</Text>
  {categories && (
    <TagList>
      {Object.entries(categories).map(([key, value]) => (
        <Tag key={key}>{value}</Tag>
      ))}
    </TagList>
  )}
</BodyContainer>
```

Reply:
Hi Tim, thank you so much for your very good review. A lot of the container are just to position things on the page. They serve no other purpose and i dont want to reuse any of them. Maybe i rename them to display their use as layout and position container?

Reply:
Yeah renaming sounds good to me! I would suggest putting it into external components anyway, just to reduce the noise from a page component but it is more due to my own preference I guess (I like my pages to have only the sections listed as components). The main point is the readability anyway so renaming might do the trick here.

Reply:
Yes i went with renaming. I added a notice to change some parts to separate components when it gets more complicated. I can definitely see the advantage by passing the prop to a details component. Will look into that again.